bin/BootstrapTypes/GenerateStmtH.cpp

```diff
@@ -123,6 +124,8 @@ void GenerateStmtH(void) {
 
       os
           << "    const void *opaque;\n"
+          << "    std::optional<::pasta::Macro> GetCoveringSubstitution() const noexcept;\n"
```

Comment:
These are in the wrong spot. Notice the `} u;` below, and if you map that to the corresponding place in `Stmt.h`, then you'll find it is in a private section of a nested `union`.  Probably you want to move it between lines 94 and 95.

Reply:
Also `(void)` for param list.

Reply:
> Probably you want to move it between lines 94 and 95.

That seems to be it, thanks